about.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,400;0,700;0,900;1,400&display=swap"
+      rel="stylesheet"
+      crossorigin="anonymous"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+      crossorigin="anonymous"
+    />
+    <script
+      src="https://kit.fontawesome.com/4c6d679277.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="about-style.css" />
+    <script src="about-script.js" defer></script>
+    <title>About Webinar</title>
+  </head>
+  <body>
+    <section class="about-section">
+      <article class="social-media">
+        <ul class="social-media-links container">
+          <li>
+            <a href=""
+              ><img src="./media/images/instagram.svg" alt="" class="icon"
+            /></a>
+          </li>
+          <li>
+            <a href=""
+              ><img src="./media/images/twitter.svg" alt="" class="icon"
+            /></a>
+          </li>
+          <li><a href="">English</a></li>
+          <li><a href="">My Page</a></li>
+          <li><a href="">Logout</a></li>
+        </ul>
+      </article>
+
+      <header>
+        <nav class="container hamburger">
+          <span class="mobile-navbar"><i class="fas fa-bars"></i></span>
+        </nav>
+        <article class="container mobile-menu d-none">
+          <nav class="open-menu">
+            <span class="close-menu-btn"><i class="fas fa-times"></i></span>
+            <ul>
+              <li><a href="index.html">Home</a></li>
+              <li><a href="about.html">About</a></li>
+              <li><a href="#">Contact</a></li>
+            </ul>
+          </nav>
+        </article>
+        <article class="desktop-menu">
+          <nav class="desktop-menu-section container">
+            <a href="index.html">
+              <img
+                src="./media/images/Logo.svg"
+                alt="Logo"
+                class="desktop-logo"
+              />
+            </a>
+            <ul class="desktop-menu-items">
+              <li><a href="about.html">About</a></li>
+              <li><a href="#">Schedule</a></li>
+              <li><a href="#">Services</a></li>
+            </ul>
+            <button href="#" class="contact-btn">Contact Us</button>
```

Comment:
- [ ] Some of menu links doesn't work like schedule link, if you still have time please implement it, this would be taken as an optional, however there is some issue Once you click on the about page there isn't another alternative of accessing home page again. please fix this by implementing the way of going back to access our home or main page instead of clicks on logo. Maybe some of users doesn't use logo link to access the main page.

![navbar2](https://user-images.githubusercontent.com/55635977/128438615-4c7b8455-8311-4692-82b4-dae93d134676.png)

Reply:
As today's the last day that's why I won't be able to work on optional pages I.e., Schedule As our main project was all about creating the Home page and About Page.

Reply:
Well its good you added a link that take you back to the homepage when you are on the about page. good job!